### PR TITLE
Fixed translation BO Emails

### PR DESCRIPTION
**Pull request by aleeks**
| Questions | Answers |
| --- | --- |
| Branch? | Fixed translation BO Emails |
| Description? | BO |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | . |
| How to test? | @AlexEven |

=> Email translations should display 2 sections, body and subjects, according to new domains and new ways of working in 1.7.
=> You should have a theme selector to modify emails from a given theme.

### CODE REVIEW

**Comment by aleeks** [comment]
Don't merge before @AlexEven 's GO

File: admin-dev/themes/default/template/controllers/translations/helpers/view/main.tpl [hunk 1]
```diff
@@ -94,11 +109,20 @@
...
+                {if $array.name}<option value="{$type}" data-controller="{if $array.sf_controller}sf{else}legacy{/if}" data-choicetheme="{$array.choice_theme}">{$array.name}</option>{/if}
             {/foreach}
           </select>
         </div>
       </div>
+      <div class="form-group" id="ps_email_selector">
+        <label class="control-label col-lg-3" for="selected-emails">{l s='Select the type of email content'}</label>
+        <div class="col-lg-4">
+          <select name="selected-emails">
+            <option value="subject" data-controller="sf">{l s='Emails subject'}</option>
```
Comment (AlexEven) [inline admin-dev/themes/default/template/controllers/translations/helpers/view/main.tpl]: Sorry but can we change to "Subject" and "Body" simply? ^^
Thank you!

Comment (aleeks) [inline admin-dev/themes/default/template/controllers/translations/helpers/view/main.tpl]: Done!

**Comment by mickaelandrieu** [comment]
Thank you @aleeks and @AlexEven

**Comment by AlexEven** [comment]
I checked and it's a go.
Thank you!

@mickaelandrieu: the wording label is a bit different from my confirming that the tests are ok - don't be too quick when merging ;)

**Comment by Shudrum** [comment]
Yep, @aleeks, please not remove the label by yourself.

**Pull request merged**